spec/calendar.html

```diff
@@ -107,6 +107,7 @@ <h1>CalendarFields ( _calendar_, _fieldNames_ )</h1>
         The CalendarFields abstract operation transforms a List of String values _fieldNames_ into another List of String values by calling the `fields` method of the given _calendar_ Object.
       </p>
       <emu-alg>
+        1. NOTE: If _calendar_ is a built-in object, then the following step will usually invoke <emu-xref href="#sec-temporal.calendar.prototype.fields"></emu-xref> or <emu-xref href="#sup-temporal.calendar.prototype.fields"></emu-xref>.
```

Comment:
I'm not opposed to adding these notes, but they are contradicted by https://github.com/tc39/proposal-temporal/pull/2485 (which could use a code review so that we can merge it :smile:) so maybe we should omit this to minimize churn?

Specifically, commit https://github.com/tc39/proposal-temporal/pull/2485/commits/6c2ce352c0476852a3496208fc3694a9721da31b#diff-1e3b606bd8b3ef5c33eaf7b43afdfc240b0a926c8d0a66e02bfc9e1a7e8de8a2 adds a step here that explicitly invokes %Temporal.Calendar.prototype.fields% if _calendar_ is a string. The current Invoke step may still invoke %Temporal.Calendar.prototype.fields% if that's the method that's found by prototype lookup (e.g., if the custom calendar extends `Temporal.Calendar` and doesn't override `fields`) but I'd no longer call it "usual".

Same thing goes for the other notes in this commit.

Reply:
Aside from the obvious benefit, this also supports convenient linking in the rendered spec. I'll remove/replace these notes after #2485 lands (and review it soon).